ui.py

```diff
@@ -101,7 +101,7 @@ def draw(self, context):
             row = layout.row()
             row.label(text="MolObject: " + obj.name)
             row = layout.row()
-            row.prop(psys, "mol_voxel_size", text="Voxel Size")
+            row.prop(psys, "mol_voxel_size", text="Resolution")
```

Comment:
Voxel size is better not to confuse with particle system grid resolution

Reply:
But particle system grid resolution _is_ what we are changing? hence why "Resolution" is better.